server/src/main/resources/reference.conf

```diff
@@ -41,6 +41,12 @@ broccoli {
     path = "./templates"
   }
 
+  # Configuration for Broccoli's Websocket connection with the webui
+  web-socket {
+    # How long to cache responses to websocket requests in Broccoli
+    cache-timeout = 3 seconds
```

Comment:
We could actually use this timeout also for caching the other requests (like service and job status) later on, right? So the good design would be to ask Nomad on demand (not every x seconds) for the job status and everything but then cache the results?

Reply:
@FRosner Yes, that's what I'd do.